core/tests/build_sources/extensions/base.py

```diff
@@ -151,25 +190,28 @@ def answer_visualizations(self):
         return result
 
     @property
-    def answer_calculation_ids(self):
+    def answer_calculation_ids(self) -> Set[str]:
         visualizations = self.answer_visualizations
         return set(
             [visualization.calculation_id for visualization in visualizations])
 
     @property
-    def dependency_ids(self):
+    def dependency_ids(self) -> List[str]:
         return copy.deepcopy(self._dependency_ids)
 
-    def normalize_answer(self, answer):
+    # Here, we used Any type for both argument and return value, because
+    # this method can accept any kind of python object and returns it's
+    # normalized form.
+    def normalize_answer(self, answer: Any) -> Any:
         """Normalizes a learner's input to this interaction."""
         if self.answer_type is None:
             return None
         else:
-            return object_registry.Registry.get_object_class_by_type(
+            return object_registry.Registry.get_object_class_by_type(  # type: ignore[no-untyped-call]
```

Comment:
same as above. Isn't this file already typed?

Reply:
Some reason as above, that's why mypy is not throwing error for these files and i missed it.

Reply:
> same as above. Isn't this file already typed?
yes, these are typed.